# EMCU diagnostics message
Header header

# Value from Roboclaw's Read Status command
uint16 status

float64 battery_voltage
float64 logic_voltage
float64 temperature

# Amperage
float64 motor1_current
float64 motor2_current
